packages/onr-core/src/mock/mirage.ts

```diff
@@ -1,22 +1,26 @@
 import { Server } from 'miragejs';
-import MOCK_JSON, { ModelsDeclaration, RoutesRegistration } from '../__mock__';
+import { getDebugger } from '@onr/common';
 
-export function makeServer({ environment = 'test' } = {}) {
+const debug = getDebugger('onr:core:mirage');
+
+export function makeServer({ environment = 'test', seeds = {}, models = {}, routes = {} } = {}) {
```

Comment:
should have more clarity type hint
for `seeds`, seems it limited the input object should have `s` in the end, how about using [Template literal type](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) to solve the problem?
and also for `models` and `routes` should be declared as `Record<string, unknown>`.

Reply:
And also, I'm not like `${string}s`  this pattern, it could be a bug when developing the mirage schema